Patch by Chris Waters sets path which are normally point to a directory
with Tk sources to a subdirectory of /usr/include/tcl8.4 (debian/rules
puts private Tk headers there).

--- a/unix/tkConfig.sh.in
+++ b/unix/tkConfig.sh.in
@@ -53,7 +53,7 @@
 
 # String to pass to linker to pick up the Tk library from its
 # build directory.
-TK_BUILD_LIB_SPEC='@TK_BUILD_LIB_SPEC@'
+TK_BUILD_LIB_SPEC='@TK_LIB_SPEC@'
 
 # String to pass to linker to pick up the Tk library from its
 # installed directory.
@@ -69,7 +69,7 @@
 # different place than the directory containing the source files, this
 # points to the location of the sources, not the location where Tk was
 # compiled.
-TK_SRC_DIR='@TK_SRC_DIR@'
+TK_SRC_DIR='@includedir@/tk-private'
 
 # Needed if you want to make a 'fat' shared library library
 # containing tk objects or link a different wish.
@@ -84,14 +84,14 @@
 
 # String to pass to linker to pick up the Tk stub library from its
 # build directory.
-TK_BUILD_STUB_LIB_SPEC='@TK_BUILD_STUB_LIB_SPEC@'
+TK_BUILD_STUB_LIB_SPEC='@TK_STUB_LIB_SPEC@'
 
 # String to pass to linker to pick up the Tk stub library from its
 # installed directory.
 TK_STUB_LIB_SPEC='@TK_STUB_LIB_SPEC@'
 
 # Path to the Tk stub library in the build directory.
-TK_BUILD_STUB_LIB_PATH='@TK_BUILD_STUB_LIB_PATH@'
+TK_BUILD_STUB_LIB_PATH='@TK_STUB_LIB_PATH@'
 
 # Path to the Tk stub library in the install directory.
 TK_STUB_LIB_PATH='@TK_STUB_LIB_PATH@'
